tools/dump-version-info.js

```diff
@@ -16,7 +16,7 @@ function getDate () {
 
 function getInfoForCurrentVersion () {
   var json = {}
-  json.version = process.versions['atom-shell']
+  json.version = process.versions['electron']
```

Comment:
Now the key doesn't have a dash I guess this can just be `process.versions.electron`